ha_lineairdb.cc

```diff
@@ -569,11 +812,76 @@ int ha_lineairdb::delete_all_rows() {
   the section "locking functions for mysql" in lock.cc;
   copy_data_between_tables() in sql_table.cc.
 */
-int ha_lineairdb::external_lock(THD*, int) {
+int ha_lineairdb::external_lock(THD* thd, int lock_type) {
   DBUG_TRACE;
+
+  thread = thd;
+  LineairDBTransaction*& tx = get_transaction(thd);
+  if (lock_type == F_UNLCK) {
```

Comment:
Add a one-line comment here on what is `F_UNLCK,` please.

Reply:
How about initializing a new local variable `const bool tx_is_ready_to_commit = lock_type == F_UNLCK;`?

Reply:
Good. It's what I've wanted.
Compiler would generate the same assembler whether it's a comment or a one-line boolean...